package.json

```diff
@@ -10,10 +10,12 @@
     "prettier-check": "prettier --check **/*.{js,json,scss,html}",
     "prettier-format": "prettier --write **/*.{js,json,scss,html}",
     "test": "yarn prettier-check && yarn workspace ssr test && yarn workspace client test",
-    "build": "cd stumptown && npm run build-json && cd .. && yarn workspace client build && yarn workspace ssr build && node cli/index.js ssr --build-html",
+    "build:content": "cd stumptown && npm run build-json && cd ..",
```

Comment:
What we really need is to know that `npm run build-json` has been run recently'ish. That's ultimately most important thing. So perhaps this line ought to become:
```
    "build:content": "cd stumptown && npm run build-json && cd .. && node cli write-content-version",
```